### PR TITLE
Fixed issue #17367: Survey policy error not show in vanilla

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: dynamically hide/show error
Dev: dynamically update  aria-describedby

### CODE REVIEW

File: themes/survey/vanilla/views/subviews/privacy/privacy_text.twig [hunk 1]
```diff
@@ -14,17 +14,48 @@
     Show the privacy message and the data security message will be available in a modal
 
 #}
-
-<!-- Privacy message -->
-<br/>
-<div class="row">
-    <div class="limit-text-window {{ aSurveyInfo.class.privacydatasectextbody }}">
-        {{ aSurveyInfo.datasecurity_notice }}
+<div class="privacy-bloc">
```
Comment (ptelu) [inline themes/survey/vanilla/views/subviews/privacy/privacy_text.twig:17]: just a small typo should be "privacy-block". Also the class is not used in any css, i assume its there for access to add customizations?

Comment (Shnoulle) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig:17]: Right for typo, and yes for adding customisation. 

Maybe i need to add again `{{ aSurveyInfo.class.privacydatasectextbody }}` ?

File: themes/survey/vanilla/views/subviews/privacy/privacy_text.twig [hunk 2]
```diff
@@ -14,17 +14,48 @@
...
-
-<!-- Privacy message -->
-<br/>
-<div class="row">
-    <div class="limit-text-window {{ aSurveyInfo.class.privacydatasectextbody }}">
-        {{ aSurveyInfo.datasecurity_notice }}
+<div class="privacy-block">
+    {% set describeprivacy = "" %}
+    {% if aSurveyInfo.datasecurity_notice %}
+        <div class="{{ aSurveyInfo.class.privacydatasectextbody }}" id="datasecurity_notice">
```
Comment (ptelu) [inline themes/survey/vanilla/views/subviews/privacy/privacy_text.twig:20]: This should get back its "limit-text-window" class. I think the "{{ aSurveyInfo.class.privacydatasectextbody }}" is fine where it is now, so I wouldn't change it.

Comment (Shnoulle) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig:20]: `<div class="limit-text-window {{ aSurveyInfo.class.privacydatasectextbody }}" id="datasecurity_notice">` ?

limit-text-window class didn't exist in css : right ?

Comment (ptelu) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig:20]: yeah looks okay, it does though themes/survey/vanilla/css/theme.css:767

Comment (Shnoulle) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig:20]: https://master.sondages.pro/256771?newtest=Y&lang=en didn't undestand readson of the border … but …

Comment (ptelu) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig]: yeah not the most optimal display but i don't want to change the design that got agreed on at some point

Comment (Shnoulle) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig]: Yes :) totally right

Comment (Shnoulle) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig]: I compare with previous to see if checkbox is inside the border.

Comment (ptelu) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig]: yeah checkbox was outside the border, just one last thing before merging could you add a padding to the "privacy" class of 20px before it had a "<br>" as first child, that gave it padding.

Comment (Shnoulle) [reply themes/survey/vanilla/views/subviews/privacy/privacy_text.twig]: Ok : same in old. i remove a br.

**Pull request merged**